packages/eslint-plugin-template/src/index.ts

```diff
@@ -56,20 +57,21 @@ export default {
   },
   processors,
   rules: {
+    [accessibilityAltTextRuleName]: accessibilityAltText,
```

Comment:
Refactors such as this (ordering things alphabetically) are only ever valuable if backed by tooling to ensure they are consistently maintained over time.

This has created unnecessary merge conflicts for other PRs, and I would not have approved it in its current state because it provides no maintainability guarantees and does not change the behaviour in any meaningful way.